### PR TITLE
support for multiple prefixes and related fixes

### PR DESCRIPTION
**Pull request by istrator2**
- I add support for multiple prefixes. Unique domains have to be used after the prefix.
- I fixed minor typos.
- My IDE did a lot of autoformatting.

I have no idea about docker, yml and other non python files, please double check them.
Maybe, more likely, something is missing.

### CODE REVIEW

**Review by krombel — APPROVED** [review]
Untested but I could not find issues through inspection.
LGTM

**Review by krombel — CHANGES_REQUESTED** [review]
Fix errors

File: .github/buildkitd.toml [hunk 1]
```diff
@@ -1,2 +1,2 @@
 [worker.oci]
-  max-parallelism = 4
```
Comment (grische) [inline .github/buildkitd.toml:2]: Ich glaube viele der Whitespace Änderungen sind unabsichtlich. Wäre besser wenn wir erst mal nichts an der CI ändern hier

**Review by grische — CHANGES_REQUESTED** [review]

File: .github/buildkitd.toml (hunk 1 above)
Comment (grische) [reply .github/buildkitd.toml]: Thanks! @istrator2

**Review by grische — APPROVED** [review]

**Comment by istrator2** [comment]
@krombel @grische: I fixed everything you asked for.

**Review by awlx — APPROVED** [review]

**Pull request merged**